members.json

```
@@ -1367,6 +1367,16 @@
 				"affiliation": "The News Corporation"
 			}
 		]
+	},
+	{
+		"name": "Anqi Li",
+		"term": [
+			{
+				"start": "2024-12-05",
+				"chair": "true",
+				"type": "appointed"
```

There aren't appointments to the AB, so this should be
>				"type": "staff-contact"

to be consistent with previous instances (E.g. Jeff Jaffe).
```suggestion
				"type": "staff-contact"
```